packages/@okta/i18n/package.json

```diff
@@ -1,27 +1,29 @@
 {
   "_args": [
     [
-      "@okta/i18n@37.1.0",
-      "/Users/mauricio.castillosilva/okta/okta-signin-widget"
+      "@okta/i18n@37.2.0-beta.2",
```

Comment:
we should promote i18n to get a release version. Just so we don't have beta version in this repo

Reply:
👍 will change